webui/src/App.js

```diff
@@ -479,22 +569,13 @@ function Plot() {
   );
 }
 
-function App() {
-  const [fetched, setFetched] = useState(false);
-  const [profiles, setProfiles] = useState([]);
-  const [activeProfile, setActiveProfile] = useState('');
-
+function FSRWebUI(props) {
```

Comment:
I renamed some components because I added a new top-level component which is now App, but I'm happy to call them something else.

Reply:
This SGTM!